CONTRIBUTING.md

```diff
@@ -258,7 +181,7 @@ in this guide.
 
 [private@pegasys.tech]: mailto:private@pegasys.tech
 [Gitter]: https://gitter.im/PegaSysEng/EthSigner
-[EthSigner documentation]: Coming soon - check docs in repo for now.
+[EthSigner documentation]: https://docs.ethsigner.pegasys.tech/en/stable/
```

Comment:
Always only point to the root of the site except if you want to point a specific content. There's a redirect and it enables us to be able to choose what version you are redirected to when you type the root url. Current redirect of the root is to /en/lates/ but we can change directly the redirect on the RDT site. If you point to /en/latest/ directly, we can't change and we have to modify the link here.
```suggestion
[EthSigner documentation]: https://docs.ethsigner.pegasys.tech/